### PR TITLE
feat(a11y): list iframes without a title

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Add the script for #5510 

## Why?

It allows us to track accessibility issues.

### CODE REVIEW

**Review by bryophyta — APPROVED** [review]
+1

File: scripts/deno/iframe-titles.ts [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+	Node,
+} from 'https://deno.land/x/deno_dom@v0.1.22-alpha/deno-dom-wasm.ts';
+import { octokit } from './github.ts';
+
+const searchParams = new URLSearchParams({
+	'api-key': 'test',
+	'query-fields': 'body',
+	q: 'iframe',
+	'show-fields': 'body',
+	'page-size': String(100),
```
Comment (AshCorr) [inline scripts/deno/iframe-titles.ts:14]: Out of curiosity, whats the difference between this and just using a string literal `'100'`?

**Review by AshCorr — APPROVED** [review]

File: scripts/deno/iframe-titles.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
+import { array, object, string } from 'https://cdn.skypack.dev/zod@3.17?dts';
+import {
+	DOMParser,
+	Element,
+	Node,
+} from 'https://deno.land/x/deno_dom@v0.1.22-alpha/deno-dom-wasm.ts';
+import { octokit } from './github.ts';
+
+const searchParams = new URLSearchParams({
+	'api-key': 'test',
```
Comment (AshCorr) [inline scripts/deno/iframe-titles.ts:10]: Is this key public knowledge already?

Comment (mxdvl) [reply scripts/deno/iframe-titles.ts:10]: Yes, it’s listed as an example in https://open-platform.theguardian.com/explore/

File: scripts/deno/iframe-titles.ts (hunk 1 above)
Comment (mxdvl) [reply scripts/deno/iframe-titles.ts:14]: I prefer the syntax highlighting that way.

**Comment by georgeblahblah** [comment]
@guardian/newsletters this might be interesting for Newsletters, a similar script could be used to find articles using old-style newsletter embeds in CAPI if you ever want to convert them to the new style.

I was also thinking it could be used in the meantime to find embeds which are improperly formatted (e.g missing the `data-component` attribute)

File: scripts/deno/iframe-titles.ts (hunk 1 above)
Comment (bryophyta) [reply scripts/deno/iframe-titles.ts:14]: this doesn't apply to the current code, but a potential gotcha that I always find a little surprising is that `new String(100)` returns something different to `String(100)`:

```js
typeof String(100) === 'string'  // true
typeof new String(100) === 'string'  // false

typeof new String(100) === 'object'  // true
```

**Comment by mxdvl** [comment]
> I was also thinking it could be used in the meantime to find embeds which are improperly formatted (e.g missing the `data-component` attribute)

We are doing exactly that for fronts in #4698, the code lives in [`scripts/deno/ophan-components.ts`](https://github.com/guardian/dotcom-rendering/blob/803644712f7a538861d68d4c336a7c83cdd8e4b9/scripts/deno/ophan-components.ts)

**Pull request merged**